sections/components.xml

```diff
@@ -399,9 +429,9 @@ in the section entitled "GNU Free Documentation License".
       linkend="com-dispatcher">Dispatcher</link> object.</para>
 
       <para>Your extension can have a custom dispatcher class,
-      <code>\My\Prefix\Dispatcher\Dispatcher</code>, where
-      <code>\My\Prefix</code> is your component's prefix. If this class
-      exists, an object instance of it is returned.</para>
+      <code>\MyComponentNamespace\Dispatcher\Dispatcher</code>, where
```

Comment:
Let's replace that here, too, with a more realistic classname.

I noticed that 3PDs reading this book copy from it (as intended!), replacing namespaces with their owns. Keeping things consistent helps them with that.